.travis.yml

```diff
@@ -1,14 +1,30 @@
 language: csharp
 
+sudo: required
+dist: trusty
+
+env:
+  - CLI_VERSION=1.0.0-preview1-002702
+
+addons:
+  apt:
+    packages:
+    - gettext
+    - libcurl4-openssl-dev
+    - libicu-dev
+    - libssl-dev
+    - libunwind8
+
```

Comment:
Looking at the core-deps Docker image it seems that they might need more packages than this, https://github.com/dotnet/dotnet-docker/blob/master/1.0.0-rc2/core-deps/Dockerfile